django_secureform/forms/__init__.py

```diff
@@ -13,10 +13,16 @@
 from django.forms.forms import NON_FIELD_ERRORS
 from django.forms.forms import BoundField
 from django.forms.forms import DeclarativeFieldsMetaclass
-from django.utils import simplejson
 from django.utils.translation import ugettext as _
 from django.utils.safestring import mark_safe
 
+try:
+    # Django <= 1.6 backwards compatibility
```

Comment:
Django <= 1.5

Reply:
It wasn't removed until [1.7](https://docs.djangoproject.com/en/2.1/releases/1.7/#features-removed-in-1-7)